src/tex/ms.tex

```diff
@@ -88,8 +89,8 @@ \subsection{Benchmark corpus}
 
 To compile our benchmark corpus, we utilized a broad list of sources (see \Cref{sec:curation}), ranging from completely novel manually crafted questions, over university exams to semi-automatically generated questions based on curated subsets of data in chemical databases.
 For quality assurance, all questions have been reviewed by at least one scientist in addition to the original curator and automated checks.
```

Comment:
For these second round we implemented the two approvals method, so I do not know if we should change this:

```suggestion
For quality assurance, all questions have been reviewed by at least two scientists in addition to the original curator and automated checks.
```